pythonforandroid/recipes/av_codecs/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pythonforandroid.toolchain import Recipe
+
+
+class PyAVCodecsRecipe(Recipe):
+    depends = ["libx264", "libshine", "libvpx"]
+
+    def build_arch(self, arch):
+        pass
+
+
+recipe = PyAVCodecsRecipe()
```

Comment:
Is this recipe doing anything at all then?

Reply:
The main intention is to keep the concept between PyAV and FFPyPlayer codecs distinct. That way, if there are any modifications to the PyAV codecs recipe, they will not interfere with FFPyPlayer.

In short, each recipe (PyAV/FFPyPlayer) in its own context.